crates/utils/src/crypto.rs

```diff
@@ -0,0 +1,13 @@
+use miden_objects::{
+    crypto::{hash::rpo::RpoDigest, rand::RpoRandomCoin},
+    Felt,
+};
+use rand::{Rng, RngCore};
+
+/// Creates a new RPO Random Coin with random seed
+pub fn get_rpo_random_coin<T: RngCore>(rng: &mut T) -> RpoRandomCoin {
+    let auth_seed: [u64; 4] = rng.gen();
+    let rng_seed = RpoDigest::from(auth_seed.map(Felt::new));
+
+    RpoRandomCoin::new(rng_seed.into())
```

Comment:
I'm surprised that `RpoRandomCoin` doesn't implement https://docs.rs/rand/latest/rand/trait.SeedableRng.html.

Feels like this is a very natural use case.

Reply:
Yes, this would simplify usage a bit. But this also would require a bit more complex conversion (from `[u8; 32]` to `RpoDigest`).